BD-1616: Add Canvas Flow steps limit (#4115)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/canvas/faqs.md

All of your existing Canvases and the original Canvas editor will continue to exist and be supported by Braze. Customers who choose to join Canvas Flow for early access will have the option of creating a Canvas using either the original or Flow workflow.

### Is there a limit for how many steps I can include?

> Yes. A Canvas built using Canvas Flow has a maximum limit of 200 steps.

Reviewer: ```suggestion
Yes. A Canvas built using Canvas Flow can contain up to 200 steps.
```
